wotlk_cat_sim.py

```diff
@@ -1708,7 +1708,11 @@ def run(self, log=False):
             # Check if a Rake tick happens at this time
             if self.rake_debuff and (time >= self.rake_ticks[0]):
                 dmg_done += self.apply_bleed_damage(
-                    self.rake_damage, 0, 'Rake', self.rake_sr_snapshot, time
+                    self.rake_damage,
+                    self.player.crit_chance * self.player.t10_4p_bonus,
```

Comment:
Crit chance for Rake should be snapshotted when it is cast, so you'll want to store the snapshotted value and use that here instead of the current crit chance. See the Rip and Lacerate tick calculations for examples of how to do this.